api/osrd_infra/schemas/path.py

```diff
@@ -18,6 +18,7 @@ class GeometryPointTrait(BaseModel):
 class RoutePath(BaseModel):
     route: ObjectReference
     track_sections: List[DirectionalTrackRange]
+    signaling_type: str
```

Comment:
Can we use an enum instead?

Reply:
yeah it can't be a string in the data model

Reply:
What would be a good solution then? I really don't think an enum is an option, this can't be edited for any new signaling module.

Reply:
We can keep it as a string here in a signaling module but enforce the type in railjson.

Reply:
But the whole point of signaling modules is that anyone can easily add their own with little to no change in the rest of the code, and this came with lots of constraints. Forcing a change in railjson with everything that implies, for any new signaling type, it removes all of this flexibility.